File: Estruturas de Repetição/Laço_doWhile.cpp
#include <iostream>
using namespace std;

int main () {
    int estoque;

    do {
        cout << "Digite a quantidade (não pode ser negativa): ";
        cin >> estoque;
    } while (estoque < 0);

    cout << "Você digitou o número correto? ";

    return 0;
}

File: Estruturas de Repetição/Laço_doWhile.java
import java.util.Scanner;

class Main{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int estoque;

        do {
            System.out.print("Digite a quantidade (não pode ser negativa): ");
            estoque = scanner.nextInt();

        } while (estoque < 0);

        System.out.println("Parabéns, você digitou o número errado ");
        scanner.close();
    }
}